packages/ice/src/turn/protocol.ts

```diff
@@ -270,9 +282,13 @@ class TurnClient implements Protocol {
     request
       .setAttribute("CHANNEL-NUMBER", channelNumber)
       .setAttribute("XOR-PEER-ADDRESS", addr);
-    const [response] = await this.request(request, this.server);
-    if (response.messageMethod !== methods.CHANNEL_BIND) {
-      throw new Error();
+    try {
+      const [response] = await this.request(request, this.server);
+      if (response.messageMethod !== methods.CHANNEL_BIND) {
+        throw new Error();
+      }
+    } catch (err) {
+      throw err
     }
```

Comment:
I think
```
catch(err) {
    throw err
}
```
might be redundant?

Reply:
I'll double check - this is where that pesky unhandled promise rejection was coming from

Reply:
right you are - have removed